### DISCUSSION TITLE
Orphaned Device Tracker Entities from official integration

### DISCUSSION THREAD

**Discussion by auanasgheps** [Vaskivskyi/ha-asusrouter#592] [General]
Hi, I am a new user of this integration and it looks awesome!
I am coming from the official one that has may issues, so I decided to switch.

- I deleted the official integration
- Installed the this integration, which requested a reboot
- Configured the new integration

The issue is that old orphaned entries are back:
![image](https://github.com/Vaskivskyi/ha-asusrouter/assets/20586878/79d91881-00ed-4154-ae6f-9720fdaed083)

Have I done something wrong? 

RT-AC86U, Merlin 386.10, latest version of HA and your integration.

**Comment by Vaskivskyi** [comment]
Hello and sorry for the late response.

These entities do not look orphaned. Why would you think so? They are just disabled (which is a default behaviour of Home Assistant for any unknown device). If you enable them, the device trackers will be showing the status of the corresponding devices

**Reply by auanasgheps** [reply]
I didn't know what I had to look for. Somebody else has suggested the correct answer when discussing another integration.
Thanks for replying!
